src/main/java/com/mineboundteam/minebound/magic/UtilitySpells/TelekineticUtilitySpell.java

```diff
@@ -40,24 +42,33 @@ public TelekineticUtilitySpell(Properties properties, TelekineticUtilitySpellCon
     public static void onPlayerTick(TickEvent.PlayerTickEvent event) {
         if (event.side.isServer() && event.phase == TickEvent.Phase.START && !event.player.isCreative()) {
```

Comment:
we also gotta check for spectator here; I tried to use spectator mode and realized I couldn't fly lol

Reply:
Ha, fixed.